index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
+    <title>Survey form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<main>
+<body>
+<h1 id="title">Survey Form</h1>
+<p id="description">Please complete this survey</p>
+<form id="survey-form">

```

Comment:
Please use class instead of ID its better to use class in css to make changes latter when needed